kubernetes/stage/secret.yaml

```diff
@@ -6,15 +6,20 @@ metadata:
+    OIDC_RP_CLIENT_ID: AgBRl2LKyI2bRwEvYROpV1te0OnVFxepz0mFxnsH43WT5vGyWgmcK4PEPDvcoA3u1PjVzEkIHcjTus/G1V9kwHKE3TRxATJXkXPxA0dEDGU1btKludY8QvLe5T6/RL0ipfjYugJdWAFb8PbJ8BpW+TmlWITcDhrPNssxioUJwTSxV4IbHgy12LXnv9Ti2//ZLMhxOq06K4bYnFQWVn+yJMoDV6HBdZmHjYQRitpwnMkDmCBQDyihcTu8Wi/lS3k8iat4Y8KSGIWXtFwP60zobSdswovCY7B0GrLWAkG6MZzSk/9pZdAZtngojWoH75EigC3q+lBbRBaswKW8v4YjBw+fWNkSjGani7rwX/rJxKb/VAmttcG0qbNabUs8WIYCjy+A/6ZnK2C+HPRTeyAQNZG8Pwgrw9L22ldPaZqrjuT1Job3I4bVQrgbkjB1ggU5W8noJ+ugvRkZs2JPJInQDE5IDUhdUQYSYfpOVVkOxmUjhZuzKYncOZpoGSncgnZkqNfe85XhBs55Mjlboa3YCH2kCsdL5q6xdAS7OJYDJf4BOmelx+27z2MkimuV/taw9K1wMmQa7QTLQEG/uiqAt6zC51fP0y9qmdAafQW0hRyxoc+w76M0Ua9zW6tNWmX2alYAfzHaBj0VYBn0PRm1LfhYx9s4FcQyVcf7UAhkjBgyMBKdlyqRre6XrONK+8KM65YaefH06CWUBohZSPiCXrxTl2C3tzimEO8Lx9onTz1BLQ==
+    OIDC_RP_CLIENT_SECRET: AgA68B4J75PCznZNORANnOmgSc7IWQoRwJFx7rcAvfZfP9zrAJ6WxsdV9WoBVA8PemRelR0AfschDK5cMJv/2YdyzPNsegj8r3yBVggFa0wXu+2S1zboDmNkZWwGffbQoK8cH++pLHeO4d3aoPSObwhBTeo4jmKXlN/DHjiIxvmBoSbC4yGrrhgVPy5lvh88yrSe5gHj/iJswPMD68K2zoPSA0jxk7ebmRFCBIf/ev8IcfF9Yh+GYT0RuFuSSWGt52qp/3QQQO9XSHDArnASESjf4k2ntyttw5PUTw/aSZatpRK/K4KzTrIWvLov93SmqLu0ldyd/VuX1cuxnssxycGLFV5Tym09vS6tYd4ASKR+BcHrphXcUaOtWf2GE2xfk/Q0Cen+G1s0E/4uD+C9J3MWDiGeYCAyeu3LNz5xdR04JaUabjJUaVmOqXn/XNp2Y2zRYflPFQoF4XtjfWgHQ11QR9AaLmR7+5AZ2Yvz9QI39O4uwXFONED9vLe7kBPT/sFCBkVbpRjE3fyUscmMKo6kAkccwrybKUU8LLZ+r1oakqVcHvqe3cpGnL0MU8Khoe3U0BJdU7An0/AyTksF6WBXjX72HMvDoWVjI8Qf+E827zXnJzqKR0XjdC0tE8OYQShEQg6O38tXdii3hb6XriHMynpY6svZ2Zkmi4v/5dZPTVyfql2dEbxMtgmEetqiBc2nX/PA8AujKrZn2gx7hFOURk9+4hs+uG0NtaXQmHKZP7wlJdYNWvewWXdoGPQB50TJnGgiSL2W5bTNhnOLq6L3
+    PASSWORD_SMTP: AgCO1sgB5IEW31thTNnh/6I/sd7XX1bnKyS0PeZ1kPXbhzMwTxk0cFC5ng5d/K4I3Msxxp/C5+MUbUNo+7sjGVrmdGNvsc73VQlgNucNQBjxHcBgGaFpqGLefaknz9NtUzM3u9/dKYESv6GwAxWkqeUEErHkJFVnQ4SaCV97NeNMaA7JyZzGu5Ehd18oSYKAUQ4R9bL7YU7RXGtKuWDkofIv5CFYEHXiD74JUfSMDpxsfWOwItu0mmm6AYLDbHw/BKt3oF/9fZ/Q2r4s+GjkKEDAol/UbZrZ19ctKXBzL2KRRfgNMqBK5CEKKRstmaMnrEFBBOfyR7hu6K17+pMHeFWbArU1/+4SrdXt65Yri9PepZF/1YzSGxs53eOwg4f2RRgI0ra/cLvb9df8fk0saL8I5akivyjrcqIAFp8cE7AjUP1aD2kl0EAQ4QvQiodAIFPvHPQ88uJt7k5IKVUu/gxKGksQg0eiRKGLcz5F+2FuakH/varrQgBfQcV2YbIBgrIrqbKa3njPS8lUQ/gEUaTCyB+Xr9bC9rl25UWbdlfNEIZ3+TFXd50eBPMcxxY8Oha5nVOZcph9CY8t0Zq3E411UJHqrb32VPloonh6AiJzm5wsZJ/tEun5uQ58nSlSwBI+8FWDEuUkVIg/ZjfeylZSlBrhEnYrTqd9qnISMBE+axJW3TWNl/C6W6zv4SbG178zSuFcHcAZJXj0LRYK/E580t8m/Ge6STVVlDLIbnX39jp8ZqitewRG2u7R3Q==
```

Comment:
super nit: I would have gone for `SMTP_USERNAME`, `SMTP_HOST` and `SMTP_PASSWORD` but if this is the order the var it's expected it's totally fine

Reply:
yeah, tbh, i thought about that too - i think i inverted bc of the samples though & no real other reason. Let me go ahead and set it straight now so it lists alphabetically in a more useful way. Thanks!

Reply:
changed, and thanks again